x/observer/types/core_params_mainnet.go

```diff
@@ -3,9 +3,81 @@
 
 package types
 
+import (
+	"fmt"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+	"github.com/zeta-chain/zetacore/common"
+)
+
 func GetCoreParams() CoreParamsList {
 	params := CoreParamsList{
-		CoreParams: []*CoreParams{},
+		CoreParams: []*CoreParams{
+			{
+				ChainId:                     common.EthChain().ChainId,
+				ConfirmationCount:           6,
+				ZetaTokenContractAddress:    "",
+				ConnectorContractAddress:    "",
+				Erc20CustodyContractAddress: "",
+				InTxTicker:                  12,
+				OutTxTicker:                 15,
+				WatchUtxoTicker:             0,
+				GasPriceTicker:              30,
+				OutboundTxScheduleInterval:  30,
+				OutboundTxScheduleLookahead: 60,
+			},
+			{
+				ChainId:                     common.BscMainnetChain().ChainId,
+				ConfirmationCount:           6,
```

Comment:
This should be higher - 14
https://help.coinbase.com/en/coinbase/getting-started/crypto-education/glossary/confirmations